Assets/PatchKit/UnityEditorExtension/Editor/Core/AppBuild.cs

```diff
@@ -75,12 +70,12 @@ public static string Location
 
             string validationError = GetLocationValidationError(value);
 
-            if (validationError == null)
+            if (validationError != null)
             {
-                return value;
+                Debug.LogError(validationError);

```

Comment:
Is logging this necessary?

If not, the whole validation checking here could be scraped.

Reply:
When value return null we don't know why. We only had information from Assert.IsNotNull. I didn't change operation for mac and linux.